#!/home/chandanha/Documents/My_Documents/Gen_AI_Udemy/Python/venv/bin/python3.12
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
